frontend/src/components/UI/FilterTags/FilterTags.jsx

```diff
@@ -0,0 +1,39 @@
+import icons from "../../../uswds/img/sprite.svg";
+
+/**
+ * A filter tags.
+ * @param {Object} props - The component props.
+ * @param {Function} props.removeFilter - A function to call to remove a filter/tag.
+ * @param {array} props.tagsList - An array of tags to display.
+ * @returns {JSX.Element} - The procurement shop select element.
+ */
+export const FilterTags = ({ removeFilter, tagsList }) => {
+    const FilterTag = ({ tag }) => (
+        <div
+            className="font-12px height-205 radius-md bg-brand-primary-light display-flex flex-align-center"
+            style={{ width: "fit-content", padding: "5px" }}
```

Comment:
we have a helper class for `fit-content` called: 
`.width-fit-content` in the `/sass/uswds/styles.scss`